spec/support/macros/define_constant.rb

```diff
@@ -28,7 +28,7 @@ def create_table(table_name, &block)
       connection.create_table(table_name, &block)
       @created_tables << table_name
       connection
-    rescue Exception => exception
+    rescue StandardError => exception
```

Comment:
Tiniest of nitpicks: perhaps rename this local variable to `error` to match?

Reply:
👍